libraries/alerts/tsconfig.lib.json

```diff
@@ -0,0 +1,19 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "outDir": "../../dist/out-tsc",
```

Comment:
Do things actually build properly with this `outDir`?

Reply:
yeah